datasets/doc/source/how-to-use-partitioners.ipynb

```diff
@@ -0,0 +1,257 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to use `Partitioner`s\n",
+    "\n",
+    "The aim of this tutorial is to make you familiar with the available `Partitioner`s that `Flower Datasets` have out-of-the-box."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# What is `Partitioner`?\n",
+    "\n",
+    "`Partitioner` is an object responsible for dividing a dataset according to a chosen strategy. There are many `Partitioner`s that you can use and all of them inherit from the `Partitioner` object which is an abstract class providing basic structure and methods that need to be implemented for any new `Partitioner` to integrate with the rest of `Flower Datasets` code. The creation of different `Partitioner` differs but the behavior is the same = they produce the same type of objects.\n",
```

Comment:
Could we add a link to the docs where we can show all the partitioners? `https://flower.ai/docs/datasets/ref-api/flwr_datasets.partitioner.html`

Reply:
Done